### ISSUE TITLE
Update globals dependency to ^15.0.0

### ISSUE THREAD

**Issue by HummingMind** [eslint/eslintrc#167]
Please update "globals" dependency to ^15.0.0

Thank you!

**Comment by nzakas** [comment]
As mentioned in the README:

> Note: This package is frozen except for critical bug fixes as ESLint moves to a new config system.

**Comment by piotr-cz** [comment]
There is a [bug](https://github.com/sindresorhus/globals/issues/244) in the globals ^14.0.0 required by this package causing eslint to fail:

```
npx eslint

Oops! Something went wrong! :(

ESLint: 9.9.1

TypeError: Key "languageOptions": Key "globals": Global "AudioWorkletGlobalScope " has leading or trailing whitespace.
    at [finalizeConfig] (xxx\node_modules\eslint\lib\config\flat-config-array.js:350:23)
    at FlatConfigArray.getConfigWithStatus (xxx\node_modules\@eslint\config-array\dist\cjs\index.cjs:1102:55)
    at FlatConfigArray.getConfig (xxx\node_modules\@eslint\config-array\dist\cjs\index.cjs:1120:15)
    at xxx\node_modules\eslint\lib\eslint\eslint-helpers.js:354:54
    at xxx\node_modules\eslint\lib\eslint\eslint-helpers.js:296:32
    at Object.isAppliedFilter (xxx\node_modules\@nodelib\fs.walk\out\readers\common.js:12:31)
    at AsyncReader._handleEntry (xxx\node_modules\@nodelib\fs.walk\out\readers\async.js:86:20)
    at xxx\node_modules\@nodelib\fs.walk\out\readers\async.js:65:22
    at callSuccessCallback (xxx\node_modules\@nodelib\fs.scandir\out\providers\async.js:103:5)
    at xxx\node_modules\@nodelib\fs.scandir\out\providers\async.js:29:13
```

It has been fixed in globals 15.0.0
